docs/api.md

```diff
@@ -70,6 +70,7 @@ These are the API group versions which are curently available when using Antrea.
 | CRD | CRD version | Introduced in | Deprecated in | Removed in |
 |---|---|---|---|---|
 | `ClusterGroup` | v1alpha2 | v1.0.0 | v1.1.0 | v1.12.0 [^1] |
+| `ClusterGroup` | v1alpha3 | v1.0.0 | v1.13.0 | N/A |
 
 [^1]: The v1alpha2 version of the `ClusterGroup` CRD is no longer served by the
       apiserver in v1.12 and is completely removed in v1.13.
```

Comment:
I wonder if it was a good idea for me to go this route. Maybe we should remove `v1alpha2` at the same time as all the other APIs we are promoting for Antrea v2.0, once we publish the CLI tool to update old API resources stored in etcd.

@tnqn @luolanzone what do you think? should we revert my change and start serving the API again?

Reply:
The v1alpha3 version has been added and used as storage version for more than 2 years, 12 minor releases. I feel it's fine to keep it as is given we haven't really deleted the version and it's still recoverable even someone is really using v1alpha2. Agree we could defer the removal to Antrea v2.0 to be safer.

Reply:
What's the decision on this? I see that `v1alpha2` was removed from the validating webhook.
I think there are 3 valid solutions:
1) revert my earlier change and handle `v1alpha2` & `v1alpha3` in the same way (remove it at the same time, and support both of them with the CLI upgrade tool)
2) keep things as they are now: version is no longer served but everything else was kept as is, and a user could update the CRD to start serving the version again
3) proceed with `v1alpha2` removal in v1.13 (including removing it from the CRD definition)

I'd feel better with option 1 as it doesn't really had any burden and is safer.

What I don't think is a worthy solution: remove support for `v1alpha2` everywhere (conversion / validation webhook) but keep the type declaration and keep the version declaration in the CRD. I feel like this solution is a strange "in-between". It's still possible for a user to set `served: true` but CRUD operations may have a strange behavior.

Reply:
1 works for me.